frontend/src/Commons/Forms/InputModules/CalculateNutrients/CalculateNutrientsList.tsx

```diff
@@ -84,7 +84,7 @@ const CalculationList: FC<CalculationListProps> = ({ field, cropBalances, result
                   <td>
                     <h4 className="col1">Fertilizer</h4>
                   </td>
-                  <td>
+                  <td aria-hidden="true">
```

Comment:
Why are you adding aria hidden here?

Reply:
It was a eslint error that's blocking me for pushing. Husky issue. Idk actually how it got past yours.